### ISSUE TITLE
Add carousel indicators

### ISSUE THREAD

**Issue by pkozlowski-opensource** [angular-ui/bootstrap#145]
There was a new release of Bootstrap's CSS and I guess that updating things should be fairly painless. 

The only change that I can immediately see is that carousel got nice indicators. There are probably other changes, for example the `container` option for tooltips and popovers, but we are discussing this in a separate issue.

Complete changelog here: https://github.com/twitter/bootstrap/blob/master/CHANGELOG.md

In short - the update should be straightforward. Didn't look at the markup for carousel indicators but I can't see reason for this to be complicated to add.
